Lib/test/test_math.py

```diff
@@ -1285,6 +1321,11 @@ def raise_after(n):
         with self.assertRaises(RuntimeError):
             sumprod(raise_after(5), range(10))
 
+        from test.test_iter import BasicIterClass
```

Comment:
Why BasicIterClass? What this class do?

Reply:
> Why BasicIterClass?  What this class do?

To not reinvent the wheel.  Here is to get StopIteration after a number of iterations (L2742, L2752).